File: Soal1/CPP/Person.cpp
#include <string>
class Person {
    protected:
        std::string nama;
        int umur;

    public:
        Person(std::string nama, int umur) {
            this->nama = nama;
            this->umur = umur;
        }
        std::string getNama() {
            return nama;
        }
        int getUmur() {
            return umur;
        }
        void setNama(std::string nama) {
            this->nama = nama;
        }
        void setUmur(int umur) {
            this->umur = umur;
        }
};

File: Soal1/Python/Person.py

class Person:
    def __init__(self, nama, umur):
        self.__nama = nama
        self.__umur = umur
    
    def getNama(self):
        return self.__nama

    def getUmur(self):
        return self.__umur
    
    def setNama(self, nama):
        self.__nama = nama
    
    def setUmur(self, umur):
        self.__umur = umur

    def __str__(self) -> str:
        return " Nama\t\t : " + self.__nama + "\n Umur\t\t : " + str(self.__umur)
